Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -204,6 +204,7 @@ @implementation FSTSpecTests {
                         userInfo:nil];
 
 - (std::unique_ptr<Persistence>)persistenceWithGCEnabled:(BOOL)GCEnabled {
+  (void)GCEnabled;
```

Comment:
From what I see in Xcode, this method is being used in two places, and there seems to be code that facilitates storing the `GCEnabled` state for the tests. However, it does not appear that `GCEnabled` is being used other than when passing it into `persistenceWithGCEnabled`, where it goes unused.

Could we just remove `GCEnabled` from this method and tests entirely, rather than having to add the `void(GCEnabled)`?